indico/modules/designer/templates/template.html

```diff
@@ -16,13 +16,12 @@
 {% endblock %}
 
 {% block content %}
-    {% if related_tpls_per_type %}
+    {% if tpls_count %}
         <div class="affected-targets-warning weak-hidden">
             {% call message_box('warning', fixed_width=true) %}
                 <p>
-                    {%- trans strong_start='<strong>'|safe, strong_end='</strong>'|safe,
-                              link_start='<a href="#" class="js-affected-objects">'|safe, link_end='</a>'|safe,
-                              count=tpls_count -%}
+                    {%- trans count=tpls_count, strong_start='<strong>'|safe, strong_end='</strong>'|safe,
```

Comment:
I'd put these two as HTML in the translated string. Moving very simple tags like `<strong>` to variables just makes the whole thing less readable with no real gain.